setup.cfg

```diff
@@ -0,0 +1,26 @@
+[flake8]
+max-line-length = 160
+exclude = out/*,venv/*,modules_archive/*,.mypy_cache/*,sessions/*
+
+[mypy]
+python_version = 3.9
+follow_imports = skip
+plugins = sqlmypy
+
+[mypy-botocore]
```

Comment:
These are all necessary to avoid mypy from looking at dependencies. Should be a better way to do this but couldn't get that working for some reason.